radio/src/telemetry/frsky.h

```diff
@@ -139,8 +139,10 @@
 #define RBOX_STATE_LAST_ID        0x0b2f
 #define RBOX_CNSP_FIRST_ID        0x0b30
 #define RBOX_CNSP_LAST_ID         0x0b3f
-#define DIY_FIRST_ID              0x5000
+#define DIY_FIRST_ID              0x5100
```

Comment:
@kilrah That would work great, and just as it always has....... i'l adjust the range for streaming